code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -279,10 +280,11 @@
 //For if the dogborg's existing patient uh, doesn't make it.
 /obj/item/device/dogborg/sleeper/proc/update_patient()
 	hound = src.loc
+
 	//Well, we HAD one, what happened to them?
 	if(patient in contents)
 		if(patient_laststat != patient.stat)
-			if(patient.stat & DEAD)
+			if((patient.stat & DEAD) || cleaning)
```

Comment:
stat is not a bitfield, don't use bitwise and. This only works coincidentally because DEAD is 2 and is a power of 2. Like if we add a constant in the future that is 6, this will return true, which would be unexpected.